### ISSUE TITLE
[Idea] Transfer search by multiple specialties

### ISSUE THREAD

**Issue by rferromoreno** [minj/foxtrick#1504]
I saw this at the forums (post 17073022.1) and I brought it here.

> There are some specialities which fit better to a certain kind of player. E.g., when I'm looking for wingers I search for headers and quick wingers only. And when it comes to very rare players I could either scan the market daily or install two player search profiles. And this is what brought me to the following idea.
>Why not making it possible to choose two specialities at once?

Currently you cannot choose multiple specialties when looking for a player in the transfer page.

So I was wondering: Why not choosing which specialties show/hide?

My first approach was this [https://github.com/minj/foxtrick/pull/1502](pull request). I added multiple check buttons to the current Foxtrick's transfer filters, so you could choose which specialties to hide on the transfer page results.

As MJ told me, using a multiple select box could be better. Also, not sure if it's better (or intuitive) to use a box for "Show" filters or for "Hide" filters (I think that a "show" instead of "hide").

**Comment by minj** [comment]
Yeah, I think 'show' selection is better:
- no spec
- head
- technical
- ...

with no selection meaning 'any', i. e. no filter

**Comment by minj** [comment]
are you still interested in implementing this?

**Comment by rferromoreno** [comment]
Yep, I'm interested but currently with no much time. I promise I will help you when I'm free, maybe in a couple of weeks.

**Comment by minj** [comment]
adding some labels to increase visibility
